codespell_lib/data/dictionary.txt

```diff
@@ -7443,6 +7448,7 @@ beastiaries->bestiaries
 beastiary->bestiary
 beatiful->beautiful
 beatifully->beautifully
+beatin->beating, beat in,
```

Comment:
⚠️ It just occurred to me we could suggest `it` in addition to `in`:
```suggestion
beatin->beating, beat in, beaten, beat it,
```
On the other hand, the probability of such a typo does seem smaller. Also, we already have multiple suggestions, so the user will have a look any way.